flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -3738,6 +3795,7 @@ class ArrayExprLowering {
     }
   }
 
+  // FIXME: should take multiple inner arguments.
```

Comment:
I do not completely follow the change with the multiple inner args. Is this only for the loops creating masks that may create several masks at once ?

Reply:
Most of the framework allows a vector of outputs to be threaded through the computational loop nest, but not here. That's not an issue presently, but we may need to be able to thread all arrays that can potentially be modified, not simply the lhs, to make the framework applicable in situations where we must track rhs effects, etc.

Reply:
OK, thanks for explaining.